packages/dcos-diagnostics/extra/dcos-diagnostics.nssm.j2

```diff
@@ -3,7 +3,7 @@ DisplayName = dcos-diagnostics
 Description = DC/OS Diagnostics For Windows Agent Node
 Application = {{ pkg_inst_dpath }}\bin\dcos-diagnostics.exe
 AppDirectory = {{ pkg_inst_dpath }}\bin
-AppParameters = --config {{ dcos_cfg_dpath }}\dcos-diagnostics\dcos-diagnostics-config.json --role=agent daemon
+AppParameters = --config {{ dcos_cfg_dpath }}\dcos-diagnostics\dcos-diagnostics-config.json --role=agent --hostname=changeme daemon
```

Comment:
This is for testing. I assume that the hostname will be `changeme` in the tests.